### ISSUE TITLE
District Builder is down

### ISSUE THREAD

**Issue by matej243** [PublicMapping/districtbuilder#1327]
## Description
I cannot do anything in District Builder

## Steps To Reproduce

1. Get the website's link
2. Go to the website
3. It will keep loading
4. Even if you refresh it will still be loading

## Expected behavior
Lead you to your maps

## Actual behavior
Loads forever, even if you refresh

## Screenshots
![image](https://github.com/user-attachments/assets/e73eb2ea-c44d-42f3-b94a-47de916667f1)

 - OS: Windows
 - Browser: Chrome

## Additional context
If you try to sign up or log in, your stuck on the sign up and log in screens.
The bug has been affecting me since yesterday now. Please fix it fast!

**Comment by samjackson24816** [comment]
I have the same issue.  When I try to create an account, I click the "Let's go!" button, and nothing happens.  I cannot get access to the site.

**Comment by matej243** [comment]
Yep, it seems like the website is down, it seems like it is facing an outage of some sorts.

**Comment by matej243** [comment]
The website went temporarily up again, but it fastly went down. Please devs, FIX THIS.

**Comment by halewis22** [comment]
I tried using the contact form to alert someone and got a 404 error. I can't find any other way to contact someone so I ended up emailing the professor who founded it at his faculty email. It's been down for almost three weeks on my end.

**Comment by CrusaderNE** [comment]
I had this error as well I thought it was just me

**Comment by halewis22** [comment]
It's now been down about five weeks for me. I think the site may be permanently dead. Nobody will respond to any of my requests for support.

**Comment by crazeeleaks** [comment]
Still having this issue. Weird. Maybe we can reach out to the orgs that fund it?

**Comment by matej243** [comment]
Since yesterday evening, District Builder seems to be working completely fine. But, i will not yet be closing this issue as i do know that this has happened many times in the past before. I have checked this morning and it is still working.

**Comment by matej243** [comment]
It seems like District Builder is down.. yet again. Well atleast it lasted for a full day!

**Comment by matej243** [comment]
> It's now been down about five weeks for me. I think the site may be permanently dead. Nobody will respond to any of my requests for support.

It's not permanently dead as far as i know. For me, every once in a while District Builder goes back up for a bit but then a few hours later closes again.

**Comment by ZeJayee** [comment]
It's still down after two months.

**Comment by Shimoregypt4227** [comment]
Has it gone down permanently? I haven't been able to use it for at least 3 months at this point.

**Comment by halewis22** [comment]
Oh yeah it’s 100% down for good. I’ve tried emailing everybody associated with it and no response from any of them. It was nice while it lasted.

**Comment by kehoops** [comment]
I have not been able to login to DistrictBuilder since early November.  I get this message: "503 Service Temporarily Unavailable".  Four months seems like more than 'temporarily' to me.  Does anyone have any information about if/when this will be up and running again?

**Comment by SomeDude4556** [comment]
its been up and down the past few weeks, like usually for an hour at a time.

**Comment by SomeDude4556** [comment]
its up currently

**Comment by SomeDude4556** [comment]
and its still up, hasn't been up that long for a while

**Comment by SomeDude4556** [comment]
well its been down since day after i posted that

**Comment by matej243** [comment]
Looks like the website is now up. I wouldn't close this issue yet, since the website does have a history of being up for a day then going down again.

**Comment by maurizi** [comment]
Hi all. I'm one the original core contributors to DistrictBuilder; I stopped working on the project shortly before I stopped working at Azavea (now Element84), who were sponsoring development.

I recently created a new version of the website, hosted at https://mikesdistrictmaker.org/

Please feel free to post any issues you find to https://github.com/maurizi/mikes-district-maker; I plan to post the code changes there once I clean it up a bit
